code/WorkInProgress/Mini/ATM.dm

```diff
@@ -340,7 +341,7 @@ log transactions
 						usr << "<span class='notice'>The [src.name] flashes an error on its display.</span>"
 						return
 					lastprint = world.timeofday
-					var/obj/item/weapon/paper/R = new(src.loc)
+					var/obj/item/weapon/paper/R = new /obj/item/weapon/paper(src.loc)
```

Comment:
Its a protip, if the type is defined in the variable and you just call new(location) like that, it takes the type of the variable!